src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -337,6 +337,41 @@ static DistributeObjectOps Procedure_Rename = {
 	.postprocess = NULL,
 	.address = RenameFunctionStmtObjectAddress,
 };
+static DistributeObjectOps Routine_AlterObjectDepends = {
+	.deparse = (void *) DeparseAlterFunctionDependsStmt,
```

Comment:
thought we didn't need `(void *)` anymore

Reply:
Slipped my mind in rebase